### PR TITLE
Create user model

### PR DESCRIPTION
**Pull request by arizade**
Added admin field with default false, established association to Reviews, and updated package.json with title and description

### CODE REVIEW

File: db/models/user.js [hunk 1]
```diff
@@ -49,4 +57,5 @@ function setEmailAndPassword(user) {
   )
 }
 
+User.hasMany(Review)
```
Comment (luisamiranda) [inline db/models/user.js]: should this go in db/index.js?

File: db/models/user.js [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
 const bcrypt = require('bcryptjs')
 const Sequelize = require('sequelize')
+
 const db = require('APP/db')
 
 const User = db.define('users', {
-  name: Sequelize.STRING,
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
```
Comment (luisamiranda) [inline db/models/user.js]: do we want to use notEmpty on all strings?

Comment (illuminatedspace) [reply db/models/user.js]: I think that's a great point.

File: db/models/user.js (hunk 1 above)
Comment (illuminatedspace) [reply db/models/user.js]: Also a great point.

**Pull request merged**